receiver/sqlserverreceiver/scraper.go

```diff
@@ -769,3 +772,197 @@ func sortRows(rows []sqlquery.StringMap, values []int64, maximum uint) []sqlquer
 	}
 	return results
 }
+
+func (s *sqlServerScraperHelper) recordDatabaseSampleQuery(ctx context.Context) (plog.Logs, error) {
+	const dbPrefix = "sqlserver."
+	// Constants are the column names of the database status
+	const DBName = "db_name"
+	const clientAddress = "client_address"
+	const clientPort = "client_port"
+	const queryStart = "query_start"
+	const sessionID = "session_id"
+	const sessionStatus = "session_status"
+	const requestStatus = "request_status"
+	const hostname = "host_name"
+	const command = "command"
+	const statementText = "statement_text"
+	const blockingSessionID = "blocking_session_id"
+	const waitType = "wait_type"
+	const waitTime = "wait_time"
+	const waitResource = "wait_resource"
+	const openTransactionCount = "open_transaction_count"
+	const transactionID = "transaction_id"
+	const percentComplete = "percent_complete"
```

Comment:
I'm trying to think of potential ways to make all of this cleaner. We could have:
```
struct attribute {
    key string
    targetType enum
}
```
We could then initialize a struct for each of these constants, and then put them all into an array. This would allow us to have a single loop looking up the key in the row, converting it to the target type, check for an error, and put the returned value into an attribute. I think that would reduce the overall complexity of this method. Do you think that would make sense?